src/server/scripts/Commands/cs_server.cpp

```diff
@@ -514,16 +514,11 @@ class server_commandscript : public CommandScript
     }
 
     // Define the 'Message of the day' for the realm
-    static bool HandleServerSetMotdCommand(ChatHandler* handler, std::string realmId, Tail motd)
+    static bool HandleServerSetMotdCommand(ChatHandler* handler, int32 realmId, Tail motd)
     {
         std::wstring wMotd   = std::wstring();
         std::string  strMotd = std::string();
 
```

Comment:
```suggestion
    static bool HandleServerSetMotdCommand(ChatHandler* handler, Optional<int32> realmId, Tail motd)
    {
        std::wstring wMotd   = std::wstring();
        std::string  strMotd = std::string();

        if (!realmId)
            realmId = -1;
```

Reply:
I don't think it is necessary to force realmId here. If none is provided we update for all realms